dds_web/templates/mail/project_release.html

```diff
@@ -16,10 +16,10 @@
         The DDS is a system for SciLifeLab infrastructures to deliver data to researchers in a fast, secure and simple
         way.</p>
       <p style="font-family: arial, sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;">
-        {% if unit_name is none %}
-        You were added to this project by {{sender_name}}.
-        {% else %}
-        You were added to this project by {{sender_name}} on behalf of {{unit_name}}{% if unit_email is not none %} (<a
+        <!-- {% if unit_name is none %} -->
+        <!-- You were added to this project by {{sender_name}}. -->
+        <!-- {% else %} -->
+        You were added to this project on behalf of {{unit_name}}{% if unit_email is not none %} (<a
           href="mailto: {{unit_email}}">{{unit_email}}</a>){% endif %}.
         {% endif %}
```

Comment:
The last `{% endif %}` doesn't have a matching `{% if %}` after the lines above are commented out, but also Project Owners can also add invite to projects so we need one case for handling unit users and one for project owners. 
```suggestion
        {% if unit_email %}
        You were added to this project on behalf of {{displayed_sender}} (<a href="mailto: {{unit_email}}">{{unit_email}}</a>).
        {% else %}
        You were added to this project by {{displayed_sender}}.
        {% endif %}
```

Reply:
> but also Project Owners can also add invite to projects so we need one case for handling unit users and one for project owners.

I realised after I wrote this that this doesn't make sense and now I understand what you meant with removing the first part.  Ignore this for now.  I'll come with a new suggestion for the project_release part.

Reply:
New suggestion: 
```suggestion
        This project was created by the following SciLifeLab unit: {{displayed_sender}} (<a href="mailto: {{unit_email}}">{{unit_email}}</a>). 
```